### ISSUE TITLE
DOTCOM-25392 - Accessibility: Author avatar has no alt text

### ISSUE THREAD

**Issue by rofe** [adobe/theblog#336]
From: https://jira.corp.adobe.com/browse/DOTCOM-25392
